infra/dev/docker-compose.yml

```diff
@@ -0,0 +1,77 @@
+version: "3.7"
+services:
+  nginx:
+    image: nginx:stable
+    container_name: dev_nginx
+    ports:
+      - 80:80
+    volumes:
```

Comment:
why isn't the `src` mapped for nginx?

Reply:
nginx talks to Laravel through a PHP-FPM connection, on another container, and thus doesn't need to have app source onboard.